src/Components/App.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react'
+import styled from 'styled-components'
+import bgImg from '../assests/bg.png'
+import Sidebar from './Sidebar'
+import Main from './Main'
+
+const App = () => {
+  return (
+    <Container>
+      <Wrapper>
+      <Sidebar/>
+      <Main/>
+      </Wrapper>
+      
```

Comment:
?

Reply:
This was made and pushed before the guidelines were given I corrected the default exports in the next pull request.